locales/en.json

```diff
@@ -1786,6 +1786,65 @@
+  "help.institution.apiUrl": "Same as Catalogue URL, if your institution exposes its records via a REST API (relevant mainly for iDigBio entries).",
+  "help.institution.institutionalGovernance": "The kind of governance (state, academia, for-profit, etc) that the institution falls under. Select a textual value from the list (see link). If none apply, choose 'OTHER', or leave empty if unknown",
+  "help.institution.disciplines": "The research disciplines the the institution houses. Select textual value(s) from the list (see link) at the most fitting level of granularity. Leave empty if none apply or unknown",
+  "help.institution.latitude": "Geographical location of the physical institution (address): decimal latitude, numeric value. Example: -28.46009",
+  "help.institution.longitude": "Geographical location of the physical institution (address): decimal longitude, numeric value. Example: -65.784278",
+  "help.institution.additionalNames": "Additional / alternative names that the institution is known under, e.g. historically or in another language",
+  "help.institution.foundingDate": "The institutions founding date (year), if known",
+  "help.institution.geographicDescription": "Geographic scope of the institution's collection holdings. Which regions of the world / countries do the collection objects (predominantly) originate from? Free-text description",
+  "help.institution.taxonomicDescription": "For biological and palaeontological collections: taxonomic scope or focus of the institution's collection holdings. Which taxon groups do the collection objects (predominantly) belong to? Free-text description",
+  "help.institution.numberSpecimens": "Estimated total number of biological / palaeontological specimens in the institution's collections, if known",
+  "help.institution.indexHerbariorumRecord": "If the information from your institution is or should be synchronized with an Index Herbariorum entry (http://sweetgum.nybg.org/science/ih/herbarium-list/): indicate yes (if no information is supplied, this defaults to 'no')",
+  "help.institution.logoUrl": "Full URL for an online image file (png, jpg) of the institutional logo, for inclusion in e.g. web pages for the institution. Leave empty if unavailable",
+  "help.institution.citesPermitNumber": "If relevant CITES permits on the regulation of trade in specimens of species (https://cites.org/eng/prog/Permit_system) are known, list the permit numbers here. Otherwise leave empty",
+  "help.institution.alternativeCodes": "In addition to the primary institution Code, above: any additional code that is already in use for the same institution in other contexts (Index Herbariorum; self-representation; references used in published collection data etc) should be identified here",
+  "help.collection.name": "The name of the institution's collection. Do not include your institution name here",
```

Comment:
Duplicate entry

Reply:
Thanks for checking. I think I am missing where the duplication is. Could you help me figure out what to remove? Thanks!